cpp_utils/BLECharacteristic.cpp

```diff
@@ -440,7 +440,7 @@ void BLECharacteristic::handleGATTServerEvent(
 		case ESP_GATTS_CONF_EVT: {
 			// ESP_LOGD(LOG_TAG, "m_handle = %d, conf->handle = %d", m_handle, param->conf.handle);
 			if(param->conf.conn_id == getService()->getServer()->getConnId()) // && param->conf.handle == m_handle) // bug in esp-idf and not implemented in arduino yet
-				m_semaphoreConfEvt.give(param->conf.status);
+				m_semaphoreConfEvt.giveFromISR();
```

Comment:
Should this be `m_semaphoreConfEvt.giveFromISR(param->conf.status);`  ?

Reply:
If you check FreeRTOS.cpp:138, the .give() variant that passes a value stores the value in m_value, which is returned by the .wait() call. Nothing is currently done with this value, so passing & storing it is irrelevant. 

Also, there is currently no '.giveFromISR()' variant that passes a value (see FreeRTOS.cpp:147).